src/lib/components/EventQAndA/index.js

```diff
@@ -0,0 +1,80 @@
+import {BaseElement} from '../BaseElement';
+import './_styles.scss';
+
+/**
+ * Filters out Q&A's based on `data-category` property.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class EventQAndA extends BaseElement {
+  constructor() {
+    super();
+    this.selectCategory = this.selectCategory.bind(this);
+
+    this.categories = new Set();
+    this.categorySelectElement = document.createElement('select');
```

Comment:
Would it make sense to define this in the same chunk of code where you're defining the div container element?

Also, since we know the prerendered children won't change, you might be able to use a `render()` function to cut down on code.

```js
render() {
  if (!this.prerenderedChildren_) {
    this.prerenderedChildren_ = [];
    for (const child of this.children) {
      this.prerenderedChildren_.push(child);
    }
  }

  return html`
    <div class="w-event-question-element">
      <select>
        <option value="">All options</option>
        ${this.categories.map(...)}
      </select>
    </div>
    ${this.prerenderedChildren_}
  `;
}
```